manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorServiceImpl.java

```diff
@@ -302,16 +317,4 @@ public ProcessorResponse toResponse(Processor processor) {
 
         return processorResponse;
     }
-
-    JsonNode definitionToJsonNode(ProcessorDefinition definition) {
```

Comment:
@kostola Does the change in serialization/deserialization impact existing entities in the database? I.e when we deploy this to `dev`, will it fail to deserialize existing entities?

Reply:
Nope it should work because the column is always `jsonb` and the payload is compatible with the `ProcessorDefinition` bean

Reply:
Thanks!